locations/spiders/neoness_fr.py

```
@@ -0,0 +1,22 @@
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class NeonessFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "neoness_fr"
+    item_attributes = {"brand": "Neoness", "brand_wikidata": "Q86668014"}
+    sitemap_urls = ["https://www.neoness.fr/robots.txt"]
+    sitemap_rules = [("/clubs/", "parse_sd")]
+    drop_attributes = {"image"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)